test/runnable/imports/ice15176b.d

```diff
@@ -0,0 +1,8 @@
+module imports.ice15176b;
+
+alias CodepointSet = InversionList!();
```

Comment:
Because the `InversionList!()` is the most frequently used non-root template instance in Phobos, so it will directly hit the current template instantiation strategy.
And, it contains an array field. Therefore its `TypeInfo` object requires `__xopEquals` and `__xtoHash` generations for the field-wise comparison and hashing. They need deferred semantic3 running, so it also hit current complex TypeInfo handling.